include/qpid/dispatch/message.h

```diff
@@ -556,12 +555,21 @@ bool qd_message_Q2_holdoff_should_unblock(qd_message_t *msg);
  */
 bool qd_message_is_Q2_blocked(const qd_message_t *msg);
```

Comment:
In some function names Q2 is capitalized, in some function names we have lower case q2 like in qd_message_is_Q2_blocked vs qd_link_q2_restart_recv

Reply:
Yeah that's a mess.  Uppercase "Q2" in a function name violates our code conventions.  I'll update the patch moving all the new names to lower case for consistency.  We can change the older names in a follow up patch.

Reply:
Fixed (for the new interfaces only)